src/layout/Group/types.d.ts

```diff
@@ -17,6 +17,7 @@ export interface IGroupEditProperties {
   openByDefault?: boolean | 'first' | 'last';
   alertOnDelete?: ExprVal.Boolean;
   saveAndNextButton?: ExprVal.Boolean;
+  alwaysShowAddButton?: ExprVal.Boolean;
```

Comment:
With the type `ExprVal.Boolean` you indicate that it's possible to use a dynamic expression here. I'm not sure if that's requested (or even needed) here, and you haven't implemented support for it in the expressions language by adding a configuration for it in `ExprConfigForGroup` (also, as far as I could see, it wasn't documented in the list of places expressions could be used).

In other words, I think the type here should be just `boolean`.